src/coordination/coordinator_state.cpp

```diff
@@ -11,19 +11,19 @@
 
 #ifdef MG_ENTERPRISE
 
-#include "coordination/coordinator_state.hpp"
+#include <optional>
+#include <string_view>
+#include <variant>
+
```

Comment:
I don't care about this but I think the usual order is:
1. header file
2. other our files
3. libraries

If you can verify with the other files, it's good if we are consistent

Reply:
I think we have incosistentices in our codebase: 
In memgraph.cpp, order of includes is:
c++ libraries
header files
3rd party libraries

Interpreter.cpp is chaos a bit. storage.cpp also. I will change as you mentioned here, I think we can raise this on our weekly meeting